skia-safe/src/core/typeface.rs

```diff
@@ -58,26 +96,64 @@ impl Default for RCHandle<SkTypeface> {
     }
 }
 
+impl PartialEq for Typeface {
+    fn eq(&self, other: &Typeface) -> bool {
+        Typeface::equal(self, other)
+    }
+}
+
 impl RCHandle<SkTypeface> {
     pub fn font_style(&self) -> FontStyle {
         unsafe { FontStyle::from_native(self.native().fontStyle()) }
     }
 
     pub fn is_bold(&self) -> bool {
-        // does not link
+        // does not link:
         // unsafe { self.native().isBold() }
-        self.font_style().weight() >= FontStyleWeight::SemiBold
+        unsafe { C_SkTypeface_isBold(self.native()) }
     }
 
     pub fn is_italic(&self) -> bool {
+        // does not link:
         // unsafe { self.native().isItalic() }
-        self.font_style().slant() != FontStyleSlant::Upright
+        unsafe { C_SkTypeface_isItalic(self.native()) }
     }
 
     pub fn is_fixed_pitch(&self) -> bool {
         unsafe { self.native().isFixedPitch() }
     }
 
+    pub fn variation_design_position(
+        &self,
+        coordinates: &mut [FontArgumentsVariationPositionCoordinate],
+    ) -> Option<usize> {
+        let r = unsafe {
+            self.native().getVariationDesignPosition(
+                coordinates.native_mut().as_mut_ptr(),
+                coordinates.len().try_into().unwrap(),
+            )
+        };
+        if r != -1 {
+            Some(r.try_into().unwrap())
+        } else {
+            None
+        }
+    }
+
+    pub fn variation_design_parameters(&self, parameters: &mut [VariationAxis]) -> Option<usize> {
+        let r = unsafe {
+            self.native().getVariationDesignParameters(
+                parameters.native_mut().as_mut_ptr(),
+                parameters.len().try_into().unwrap(),
+            )
+        };
+        if r != -1 {
+            Some(r.try_into().unwrap())
+        } else {
+            None
+        }
+    }
+
     pub fn equal(face_a: &Typeface, face_b: &Typeface) -> bool {

```

Comment:
This function should probably be removed, it's confusing to have both `PartialEq::eq` and `Typeface::equals` at hand.

Reply:
I've decided to keep it for now and remove the `PartialEq` implementation, because Skia does not implement `operator ==` for some reason.